static/redfish.yaml

```diff
@@ -1022,6 +1094,8 @@ paths:
   /TaskService/Oem/Tasks/{identifier}:
     x-swagger-router-controller: task-service
     get:
+      x-privileges: [ 'Login' ]
```

Comment:
For now there is a hardcoded set or role to privileges here:https://github.com/RackHD/on-http/pull/178/files#diff-a8be6600959fd2bbeb19f645072425c5R57.  As for where they came from, the Redfish ones are defined in the spec and the 2.0 ones are a stab at something to get it going.

The 'roles' model is added to on-core with the intention that a user will use an API (after we create it) to define their own set of Role/Privilege mappings.  There needs to be a continuing effort to refine the 2.0 privileges to something more granular than Read/Write.